Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

```diff
@@ -293,3 +293,5 @@ spec:
        sum(rate(ceph_rgw_put[2m])) # promethues query used for autoscaling
      threshold: "90"
 ```
+
+P.S During reconciliation of `cephobjectstore`, the Rook Operator will reset the replica count for RGW which was set by horizontal pod scaler. And when it identifies horizontal pod scaler revert the change again. This is briefly discussed (here)[https://github.com/rook/rook/issues/10001]
```

Comment:
```suggestion
P.S During reconciliation of a CephObjectStore, the Rook Operator will reset the replica count for RGW which was set by horizontal pod scaler. The horizontal pod autoscaler will change the again once it re-evaluates the rule. This can result in a performance hiccup of several seconds after a reconciliation. This is briefly discussed (here)[https://github.com/rook/rook/issues/10001]
```